src/platform_impl/macos/mod.rs

```diff
@@ -1,4 +1,5 @@
 #![cfg(target_os = "macos")]
+#![allow(clippy::let_unit_value)]
```

Comment:
That we have to specify this is actually a regression, I've filed an issue about it here: https://github.com/rust-lang/rust-clippy/issues/8998.